docs/security-champion/faq.md

```diff
@@ -6,7 +6,19 @@
 ### Who can become a Security Champion?
 Everyone who considers themselves part of a development team can become a Security Champion. If you are a developer, ux-designer, tester, citizen developer or anything in-between, you are welcome to join. There is no requirement to be an Equinor employee to join, we invite consultants as well!
 
-### I want to attend the Kick off event in october. Do you provide a WBS for hours and travel expenses?
+### Does being a Security Champion result in a lot of extra work?
+It depends on what _you_ want to do. It can be everything from just informing the team about security related issues/questions you hear about in the network to facilitate regular threat modeling sessions, implement Snyk in your pipelines, performing pen-testing (i.e. hack your solutions) and a ton of
```

Comment:
```suggestion
It depends on what _you_ want to do. It can be everything from just informing the team about security related issues/questions you hear about in the network, to facilitating regular threat modelling sessions, or implementing Snyk in your pipelines, and a ton of
```
Some syntax adjustments, and I have removed pen-testing since we don't do that actively today